src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1722,6 +1722,7 @@
     Create a webapp and deploy code from a local workspace to the app. The command is required to run from the folder
     where the code is present. Current support includes Node, Python, .NET Core and ASP.NET. Node,
```

Comment:
Update the help text "Current support includes Node, Python, .NET Core and ASP.NET. Node" to say and static html.